main.go

```diff
@@ -0,0 +1,226 @@
+package main
+
+import (
+	"archive/zip"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-tools/go-steputils/input"
+)
+
+const (
+	targetExtension string = ".zip"
+)
+
+// ConfigsModel ...
+type ConfigsModel struct {
+	SourcePath string
+	TargetPath string
+}
+
+func main() {
+	configs := createConfigsModelFromEnvs()
```

Comment:
wire in our new config handling: https://github.com/bitrise-steplib/steps-github-status/blob/master/main.go#L103